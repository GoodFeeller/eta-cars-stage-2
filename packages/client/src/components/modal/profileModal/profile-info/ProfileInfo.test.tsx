import { render, screen } from "@testing-library/react"
import ProfileInfo from "./ProfileInfo"

describe('Profile info test', () => {
    test('Text must be black snapshot', () => {
        render(<ProfileInfo nowPrice={100} oldPrice={100}/>)
        const info = screen.queryAllByTestId('profile-info')
        expect(info).toMatchSnapshot()
    })
    test('Text must be red snapshot', () => {
        render(<ProfileInfo nowPrice={50} oldPrice={100}/>)
        const info = screen.queryAllByTestId('profile-info')
        expect(info).toMatchSnapshot()
    })
    test('Text must be green snapshot', () => {
        render(<ProfileInfo nowPrice={150} oldPrice={100}/>)
        const info = screen.queryAllByTestId('profile-info')
        expect(info).toMatchSnapshot()
    })
})