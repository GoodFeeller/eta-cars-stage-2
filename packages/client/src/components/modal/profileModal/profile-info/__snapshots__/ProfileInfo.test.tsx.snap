// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile info test Text must be black snapshot 1`] = `
[
  <div
    class="infoBlock"
    data-testid="profile-info"
  >
    <span
      class="profilePrice"
    >
      Total price:
       
      $100.00
    </span>
    <span
      class="changes"
      style="color: black;"
    >
      $0.00
    </span>
    <span
      style="color: black;"
    >
      0%
    </span>
  </div>,
]
`;

exports[`Profile info test Text must be green snapshot 1`] = `
[
  <div
    class="infoBlock"
    data-testid="profile-info"
  >
    <span
      class="profilePrice"
    >
      Total price:
       
      $150.00
    </span>
    <span
      class="changes"
      style="color: green;"
    >
      $50.00
    </span>
    <span
      style="color: green;"
    >
      33.33%
    </span>
  </div>,
]
`;

exports[`Profile info test Text must be red snapshot 1`] = `
[
  <div
    class="infoBlock"
    data-testid="profile-info"
  >
    <span
      class="profilePrice"
    >
      Total price:
       
      $50.00
    </span>
    <span
      class="changes"
      style="color: red;"
    >
      -$50.00
    </span>
    <span
      style="color: red;"
    >
      -100%
    </span>
  </div>,
]
`;
